modular_citadel/code/game/objects/items/meat.dm

```diff
@@ -0,0 +1,26 @@
+/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/ipc
+	icon = 'modular_citadel/icons/obj/foods.dmi'
+	icon_state = "ipcmeat"
+	desc = "Gross robot meat."
```

Comment:
My meat's a cultural delicacy! Ree!

Reply:
I strongly suggest just making them gib into metal sheets/plasteel instead of having them generate meat at all.